enos/enos.py

```diff
@@ -185,9 +210,14 @@ def install_os(env=None, **kwargs):
     else:
         kolla_cmd.append('deploy')
 
-    kolla_cmd.extend(["-i", "%s/multinode" % SYMLINK_NAME,
-                      "--passwords", "%s/passwords.yml" % SYMLINK_NAME,
-                      "--configdir", "%s" % SYMLINK_NAME])
+    # Override default result directory name when --env parameter is given
```

Comment:
This piece of code seems duplicated 4 times. Is it possible to factorize it somehow ?